src/contentmanager.h

```diff
@@ -45,6 +45,7 @@ class ContentManager : public QObject
     void booksChanged();
     void downloadsChanged();
     void currentLangChanged();
+    void displayLoadIcon(const bool);
```

Comment:
I wonder if we could name it differently.
`displayLoadIcon` is more about what to do with this signal. But the signal should be about what is going on.
Something like `pendingRequest` or `Loading`. It would be to the slot of the signal to decide what to do when a request is pending (display a loading icon).